popup.js

```diff
@@ -44,6 +49,7 @@ nexts.forEach((next) => {
     show(index);
   };
 });
+
 previouses.forEach((previous) => {
   previous.onclick = () => {
     index -= 1;
```

Comment:
the variable name 'previouses' could be substituted to previousButtons. Make it more descriptive.